backend/execution_cache/license_intermediate_data/additional_license_info.csv

```diff
@@ -0,0 +1,18 @@
+https://github.com/docker/spdystream/blob/master/LICENSE,Apache License 2.0
```

Comment:
same here. please follow backend conventions for handling the license files. example
https://github.com/kubeflow/pipelines/blob/master/backend/Dockerfile.persistenceagent#L16

Reply:
Can you please point to the tool that keeps the licenses.txt up to date when the GO package dependencies change?